account_invoice_inter_company/views/account_move_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field
+            name="name"
+        >account.move.form - HACK: Extra fields for odoo.tests.Form() to work</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']/tree" position="inside">
+                <field name="company_id" invisible="0" readonly="0" />
```

Comment:
Always visible???

Reply:
It's a primary for this specific purpose, so it shouldn't be a problem, isn't it?

Reply:
Indeed, although I think I wanted to put invisible in that moment and it was a typo. It doesn't matter anyway.